<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QMainWindow" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Podcast Downloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>RSS Feed</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEditRSS"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Output Folder</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButtonDownload">
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEditFolder"/>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButtonFolder">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEditFilter">
      <property name="text">
       <string>^.((?!filter).)*$</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QTreeView" name="treeView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
